pkg/skaffold/runner/verifier.go

```diff
@@ -52,8 +52,16 @@ func GetVerifier(ctx context.Context, runCtx *runcontext.RunContext, labeller *l
 		}
 	}
 
+	var defaultDeployNS *string
+	if runCtx.DefaultPipeline().Deploy.KptDeploy != nil {
+		defaultDeployNS = runCtx.DefaultPipeline().Deploy.KptDeploy.DefaultNamespace
+	}
+	if runCtx.DefaultPipeline().Deploy.KubectlDeploy != nil {
+		defaultDeployNS = runCtx.DefaultPipeline().Deploy.KubectlDeploy.DefaultNamespace
+	}
```

Comment:
I was thinking, does it make sense to use the `runCtx.GetNamespace` method here? I know we are not using the `DefaultPipeline` there, instead we are going through all the pipelines to calculate the namespace, but maybe is ok to have the same behaviour, what do you think?